internal/feature/features_test.go

```diff
@@ -53,13 +53,21 @@ func TestContext(t *testing.T) {
 	t.Parallel()
 	gate := NewGate()
 	ctx := NewContext(context.Background(), gate)
```

Comment:
🤔 This seems right to me in this moment. Time to step away!

```go
	assert.Equal(t, ShowAssigned(ctx), "")
	assert.Assert(t, ShowEnabled(ctx) != "") // This assumes some feature is enabled by default.

	assert.NilError(t, gate.Set("TablespaceVolumes=true"))
	assert.Assert(t, Enabled(ctx, TablespaceVolumes))
	assert.Equal(t, ShowAssigned(ctx), "TablespaceVolumes=true")
	assert.Assert(t,
		strings.Contains(ShowEnabled(ctx), "TablespaceVolumes=true"),
		"got: %v", ShowEnabled(ctx))

	assert.NilError(t, gate.SetFromMap(map[string]bool{TablespaceVolumes: false}))
	assert.Assert(t, !Enabled(ctx, TablespaceVolumes))
	assert.Equal(t, ShowAssigned(ctx), "TablespaceVolumes=false")
	assert.Assert(t,
		!strings.Contains(ShowEnabled(ctx), "TablespaceVolumes"),
		"got: %v", ShowEnabled(ctx))
```

Reply:
Yeah, as we graduate feature gates, we'd have to keep updating the test.